### ISSUE TITLE
Module "jstore" fails to load cl_texturelocs.lua

### ISSUE THREAD

**Issue by wertercatt** [Foohy/jazztronauts#156]
```
[jazztronauts] Couldn't include file 'cl_texturelocs.lua' - File not found (@addons/jazztronauts/gamemodes/jazztronauts/gamemode/store/cl_init.lua (line 3))
  1. unknown - addons/jazztronauts/gamemodes/jazztronauts/gamemode/store/cl_init.lua:3
   2. include - [C]:-1
    3. unknown - addons/jazztronauts/gamemodes/jazztronauts/gamemode/cl_init.lua:5
```

It was moved out of the module in https://github.com/Foohy/jazztronauts/commit/c7e1f3f79f3aac04421fbdcfc2f7beca5ef233c0

There was a reason it was in there @Foohy 😛

**Comment by Foohy** [comment]
Doh!! Thanks for the catch that's what I get for rushing, I'll fix that here in a second.

**Comment by Foohy** [comment]
Fixed in fbae848e0b0076a6c8a6105e0a52269d46d66880
